### ISSUE TITLE
Suggestion: Add an Tag to all fillable bookshelves

### ISSUE THREAD

**Issue by TheButterbrotMan** [chimericdream/minekea-fabric#7]
I would reall appriciate when you could add a tag to all container bookshelves like ```shelf_storage:shelf_storage```

**Comment by chimericdream** [comment]
That's a good idea. I'll try to (remember to) get it into the next release. :-)

**Comment by chimericdream** [comment]
This is included in the 2.1.0 release! I added the following tags:

```
minekea:bookshelves
minekea:bookshelves_with_storage
minekea:crates
```
